### PR TITLE
Updated Italian Translation

### PR DESCRIPTION
**Pull request by maicol07**
Work in progress...

### CODE REVIEW

File: app/src/main/res/values-it/strings.xml [hunk 1]
```diff
@@ -342,4 +342,519 @@
     <string name="successfully_invited">Invitato con successo</string>
     <string name="reply">Rispondi</string>
     <string name="failed_selecting_image">Caricamento immagine fallito.</string>
+    <string name="private_repo">Privato</string>
+
```
Comment (yakov116) [inline app/src/main/res/values-it/strings.xml]: Why are you adding a new line in between each string?

Comment (maicol07) [reply app/src/main/res/values-it/strings.xml]: @yakov116 I simply copied the strings from the English file... but in the next update I'll remove them!

_Sent from my ZenFone 3 (ZE520KL) using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

**Comment by maicol07** [comment]
@dedepete Fixed!

_Sent from my ZenFone 3 (ZE520KL) using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

**Comment by dedepete** [comment]
Same thing with `rect_avatar_summary`.

**Comment by k0shk0sh** [comment]
Please rebase all your commits into one commit after you done otherwise we gonna end up with plenty of commits from one pr. 

Thanks

**Comment by yakov116** [comment]
@k0shk0sh we can just squash and merge they pr

**Comment by maicol07** [comment]
Finished translation. Check if it's OK. Also, are new strings added in the meantime (from 1st commit)?

**Comment by maicol07** [comment]
Finished translation. Check if it's OK. Also, are new strings added in the meantime (from 1st commit)?

**Comment by dedepete** [comment]
Latest changes were made on Jan 20, 2018. You can browse the repo yourself to find out.

**Comment by yakov116** [comment]
@maicol07 please go through the file and remove all the duplicates.

**Comment by dedepete** [comment]
@maicol07 are you still on it? If you want we can help to clean the things up.

**Comment by maicol07** [comment]
@dedepete Yes, but from mobile is difficult to search duplicates. Also, when I have time I could check on my PC. Thanks for your help!

_Sent from my ZenFone 3 (ZE520KL) using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

**Comment by maicol07** [comment]
Hi, sorry if I've gone away for long time... but because of school I couldn't continue the translation. Can you please help me fixing the errors (like spaces and duplicated strings) and adding new strings not translated?
Also are the strings related to a banner feature necessary?
Thanks

**Comment by maicol07** [comment]
Hi, I'm actually updating the translation with all the fixes and with all the new strings, but should I remove this:
```
<resources xmlns:tools="http://schemas.android.com/tools"
           tools:ignore="MissingTranslation">
```
Thanks

**Comment by maicol07** [comment]
Finally reviewed and updated!! It's ready to merge!!

**Comment by cozyplanes** [comment]
@k0shk0sh

**Comment by yakov116** [comment]
I will go through it soon

**Comment by yakov116** [comment]
@dedepete let me go though it first

**Comment by dedepete** [comment]
@yakov116 np.

_Отправлено с моего P2 с помощью [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

**Pull request merged**